cpp/src/barretenberg/proof_system/flavor/flavor.hpp

```diff
@@ -1,8 +0,0 @@
-#pragma once
```

Comment:
We don't need any of this, but I didn't delete the file since it will be needed soon.